.circleci/config.yml

```diff
@@ -138,9 +138,68 @@ jobs:
             pip install codecov && codecov \
               || echo "Error in uploading coverage reports to codecov.io."
 
+  "bionic-python-3.6":
+    docker:
+      # This is built from tools/circleci/images/bionic/Dockerfile .
+      - image: hackerkid/circleci:bionic-python-5.test
```

Comment:
@gnprice what should this image be changed to?  

I'm fine with merging after we update that.

Reply:
Should be `gregprice/circleci:bionic-python-1.test`. (It's the latest/only "bionic" image on https://hub.docker.com/r/gregprice/circleci/tags/ .)

I'd wait for a test run on a branch, though, just to double-check that there isn't some typo in that syntax and also that my build of the image didn't somehow go wrong.